src/util/psbt/serialize.rs

```diff
@@ -20,7 +20,7 @@
 
 use prelude::*;
 
-use io;
+use ::{EcdsaSig, io};
```

Comment:
nit: Non-merge blocking. 

I don't like merging std imports in the same line as same library imports.

Reply:
Oh, this is IDE's work. Will fix if there will be another code update round